### DISCUSSION TITLE
Wrong Step number in Github Page about to fork a repo

### DISCUSSION THREAD

**Discussion by dammi01** [community/community#14639] [Other Feature Feedback, Questions, & Ideas]
I've found a little mistake in the explaination about how to fork a repo.  In step 7 where it explains how to fork the repo and sync it on yours , it says --> Type git remote add upstream, and then paste the URL you copied in Step 2 and press Enter.
I think it should be Step 3 and not Step 2. Confirm ? Here's the link of the page: https://docs.github.com/en/get-started/quickstart/fork-a-repo

**Comment by look** [comment]
Thanks for the report!

I opened a PR to fix this: https://github.com/github/docs/pull/17004

 FYI, our documentation is open source so if you see similar issues in the future you can open PRs too. :shipit:
